Word.py

```diff
@@ -0,0 +1,53 @@
+import nltk
+from nltk.corpus import wordnet
+
+
+class Word:
+
+    def __init__(self, word_text):
+        self.text = word_text
+        self.synonyms = []
+        self.antonyms = []
+        self.wnset = ""
+        self.pos = ""
+        self.definitionList = {}
+        self.getPOS(self)
+        self.getSynAnt(self)
+
+    #gets the nltk POS and then uses the mapper function to map to wordnet POS
+    @staticmethod
+    def getPOS(word):
```

Comment:
change to get_pos
Apparantly all python methods are supposed to be all lower case